GenumKit/Parsers/StoryboardParser.swift

```diff
@@ -109,26 +123,43 @@ public final class StoryboardParser {
       }
     }
   }
+
+  private func collectModules(initial: InitialScene?, scenes: Set<Scene>, segues: Set<Segue>) -> Set<String> {
+    var result = Set<String>()
+
+    if let module = initial?.customModule {
+      result.insert(module)
+    }
+    result.formUnion(Set<String>(scenes.flatMap { $0.customModule }))
+    result.formUnion(Set<String>(segues.flatMap { $0.customModule }))
+
+    return result
+  }
 }
 
 extension StoryboardParser.Scene: Equatable { }
 func == (lhs: StoryboardParser.Scene, rhs: StoryboardParser.Scene) -> Bool {
-  return lhs.storyboardID == rhs.storyboardID && lhs.tag == rhs.tag && lhs.customClass == rhs.customClass
+  return lhs.storyboardID == rhs.storyboardID &&
+    lhs.tag == rhs.tag &&
+    lhs.customClass == rhs.customClass &&
+    lhs.customModule == rhs.customModule
 }
 
 extension StoryboardParser.Scene: Hashable {
   var hashValue: Int {
-    return "\(storyboardID);\(tag);\(customClass)".hashValue
```

Comment:
btw I wonder if it wouldn't be better to `[storyboardID,tag,customModule,customClass].map({ $0.hashValue }).reduce(0, ^)`?